buildSrc/src/main/java/com/uber/okbuck/core/model/jvm/JvmTarget.java

```diff
@@ -18,6 +18,10 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 import javax.annotation.Nullable;
+
+import kotlin.collections.CollectionsKt;
```

Comment:
Please avoid kotlin stdlib and use standard ja a stream apis instead

Reply:
Looks like these are just stale imports from when I was testing. Though comment for the future - what I was doing that used this is hardly possible with streams APIs, they're not 1:1